pkg/apis/hive/v1alpha1/syncset_types.go

```diff
@@ -79,21 +81,17 @@ type SyncObjectPatch struct {
 type SyncConditionType string
 
 const (
-	// CreateSuccess indicates whether the resource has been created or not. If not,
-	// it should include a reason and message for the failure.
-	CreateSuccess SyncConditionType = "CreateSuccess"
-
-	// UpdateSuccess indicates whether the resource has been updates or not. If not,
-	// it should include a reason and message for the failure.
-	UpdateSuccess SyncConditionType = "UpdateSuccess"
-
-	// PatchSuccess indicates whether the patch has been applied or not. If not,
-	// it should include a reason and message for the failure.
-	PatchSuccess SyncConditionType = "PatchSuccess"
+	// ApplySuccess indicates whether the resource or patch has been applied or not.
+	// If not, it should include a reason and message for the failure.
+	ApplySuccess SyncConditionType = "ApplySuccess"
```

Comment:
append `SyncCondition` to each condition variable so we can distinguish from other constants in the hivev1 package
so s/ApplySuccess/ApplySuccessSyncCondition/